.circleci/config.yml

```diff
@@ -268,7 +268,6 @@ jobs:
             export COMMIT="${CIRCLE_SHA1:8:8}"
             export CONTAINER_TAG="${COMMIT}"
             # export CONTAINER_FORCE_PULL="true"
-            ./scripts/install-kubectl.sh
```

Comment:
Is this line removal is required?

Reply:
There is no point for calling the script here since it is the first thing called in `./scripts/prepare-circle-integration-tests.sh` a line later.